LetsDoStuff.WebApi/Controllers/ActivityController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using LetsDoStuff.Domain;
+using LetsDoStuff.Domain.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace LetsDoStuff.WebApi.Controllers
+{
+    [Route("api/activities")]
+    public class ActivityController : ControllerBase
+    {
+        private LdsContext db;
+
+        public ActivityController(LdsContext context)
+        {
+            db = context;
+        }
+
+        /// <summary>
+        /// Get list of activities.
+        /// </summary>
+        /// <returns>All activities.</returns>
+        [HttpGet]
+        public ActionResult<ICollection<Activity>> GetActivityOutput()
+        {
+            var activities = db.Activities.Include(a => a.Tags).Include(a => a.Creator).ToList();
+
+            return activities;
+        }
+
+        /// <summary>
+        /// Get a specific activity.
+        /// </summary>
+        /// <param name="id">ID of activity.</param>
+        /// <returns>A specified activity.</returns>
+        [HttpGet("activity")]
+        public ActionResult<Activity> GetActivity(int id)
+        {
+            var activity = db.Activities.Include(a => a.Tags).Include(a => a.Creator).FirstOrDefault(itm => itm.Id == id);
```

Comment:
I am not sure we need eager logic here. Let's check if it works without Includes just in case

Reply:
Without Includes GET request always returns activities with null creator and empty list of tags